### ISSUE TITLE
Info Command (json): History ID incorrect datatype

### ISSUE THREAD

**Issue by jasonodonnell** [pgbackrest/pgbackrest#421]
When you output backup info using `json` format, the history ID is returned as a string:

```json
"db" : [
    {
	"id" : "1",
	"system-id" : 6475061310255132704,
        "version" : "9.6"
    }
],
```

The [source](https://github.com/pgbackrest/pgbackrest/blob/1e0ed074555df7d9079fdf52c4b6b5b049b6d155/lib/pgBackRest/Info.pm#L505) suggests this should be an integer but I cannot locate where its being casted to a string.

**Comment by jasonodonnell** [comment]
Possibly the same issue with the `archive` blob as well:

```json
"archive" : [
	 {
	     "database" : {
	          "id" : "1"
	      },
	      "id" : "9.6-1",
	     "max" : "00000001000000000000000A",
             "min" : "000000010000000000000001"
    }
],
```

**Comment by dwsteele** [comment]
Well, I'm not completely sure which one is right but it should definitely be all one way or the other.  @cmwshang, what do you think?

**Comment by cmwshang** [comment]
Agreed it should all be one or the other, but it appears to always be a string in the output even though the variable prefix in the code is $i.  At this point the data is read from the file so it's just character strings - we should be able to type it truly to an integer if we do the + zero trick. Is this causing an error somewhere?

Cynthia Shang
www.crunchydata.com

**Comment by jasonodonnell** [comment]
Agreed, if its an integer, it should be an integer.

I noticed this inconsistency when trying to auto generate golang structures of the json output.  The mix-match of data types resulted in a more complex object.

I've seen no errors related to this, just pointing it out for the sake of consistency.

**Comment by cmwshang** [comment]
Sounds good. I'll make the change for next release.

Cynthia Shang
www.crunchydata.com

**Comment by dwsteele** [comment]
Fixed at 2d56de00fcb04f2f949c18.
